<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caesar</class>
 <widget class="QMainWindow" name="Caesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Caesar - Program for encoding and decoding Caesar cipher</string>
  </property>
  <property name="windowIcon">
   <iconset theme="CaesarMaterialTheme">
    <normaloff>resources/key-icon.png</normaloff>resources/key-icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="TextLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Поле для ввода</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Ваш исходный текст должен быть здесь..</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Поле для вывода преобразованного текста</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Здесь будет отображаться обработанный текст</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="ButtonsLayout">
      <item>
       <widget class="QPushButton" name="EncodeButton">
        <property name="statusTip">
         <string>Зашифровать текст</string>
        </property>
        <property name="text">
         <string>Зашифровать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="shiftLabel">
        <property name="text">
         <string>Сдвиг:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shiftSpinbox">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Количество символов алфавита, на которое будет сдвинута каждая буква слова</string>
        </property>
        <property name="minimum">
         <number>-99</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="DecodeButton">
        <property name="statusTip">
         <string>Декодировать текст</string>
        </property>
        <property name="text">
         <string>Дешифровать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AlgoDecodeCheckbutton">
        <property name="toolTip">
         <string>Программа будет расшифровывать текст методом частотного анализа (иначе - перебором)</string>
        </property>
        <property name="statusTip">
         <string>Программа будет расшифровывать текст методом частотного анализа (иначе - перебором)</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Алгоритмически дешифровать</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LanguageLayout">
        <item>
         <widget class="QLabel" name="languageLabel">
          <property name="text">
           <string>Язык:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RussianLanguageRadio">
          <property name="text">
           <string>Русский</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="EnglishLanguageRadio">
          <property name="text">
           <string>Английский</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
